src/iris/api.py

```diff
@@ -700,6 +706,19 @@ def process_resource(self, req, resp, resource, params):  # pragma: no cover
         if resource.allow_read_no_auth and method == 'GET':
             return
 
+        # Ignore HMAC requirements for custom webhooks
+        if req.env['PATH_INFO'].startswith('/v0/webhooks/'):
+            app_name = req.get_param('application', required=True)
+            app = cache.applications.get(app_name)
```

Comment:
really sorry to ask for another change but the `if not app` check should still be present here to make sure the one we did get from the query string is present in the applications dict.